specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/engine.schema.json

```diff
@@ -70,6 +70,19 @@
             },
             "additionalProperties": false
         },
+        "schemas":
```

Comment:
I'm not sure if `schemas` is actually the right word here, since `schema` actually represents the types. not the system holding them.

The closest I have found for a collection of schemas is `catalog`.
https://stackoverflow.com/questions/7022755/whats-the-difference-between-a-catalog-and-a-schema-in-a-relational-database/7023159#7023159

This would give us `SchemaRegistryCatalog`, etc as we provide implementations.

What do you think?